### PR TITLE
docs: python api review

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/api/class-browsercontext.md [hunk 1]
```diff
@@ -219,6 +225,7 @@ Whether to pass the argument as a handle, instead of passing by value. When pass
 supported. When passing by value, multiple arguments are supported.
 
 ## async method: BrowserContext.exposeFunction
+**DEPRECATED** Use [`method: BrowserContext.exposeBinding`] instead.
```
Comment (mxschmitt) [inline docs/src/api/class-browsercontext.md:228]: lets remove it maybe from Python directly?

File: types/types.d.ts [hunk 2]
```diff
@@ -415,9 +419,12 @@ export interface Page {
    * Emitted when attachment download started. User can access basic file operations on downloaded content via the passed
    * [Download] instance.
    * 
-   * > **NOTE** Browser context **must** be created with the `acceptDownloads` set to `true` when user needs access to the
-   * downloaded content. If `acceptDownloads` is not set or set to `false`, download events are emitted, but the actual
-   * download is not performed and user has no access to the downloaded files.
+   * :::note
```
Comment (mxschmitt) [inline types/types.d.ts:422]: this `:::note` isn't official markdown syntax, so shall we maybe transform it for the types and api.json so editors like VSC can display it correctly?

File: docs/src/api/class-browsercontext.md (hunk 1 above)
Comment (pavelfeldman) [reply docs/src/api/class-browsercontext.md:228]: We actually considered it and decided to leave it as is. I'll remove the deprecation note.

File: types/types.d.ts (hunk 2 above)
Comment (pavelfeldman) [reply types/types.d.ts:422]: Yeah, we should do it at a codegen stage, was planning on doing it later, but might as well do it right away.

**Pull request merged**